requirements.txt

```diff
@@ -77,7 +77,7 @@ panda3d==1.10.7
 panda3d-gltf==0.10
 panda3d-simplepbr==0.7
 pandas==1.1.3
-Pillow==8.0.0
+Pillow==7.2.0
```

Comment:
This should fix the problem that the ULTRA team is having with this dependency right?

Reply:
Yes, I have updated the requirements to:
+ Pillow==7.2.0
+ chardet==3.0.2